internal/confmapprovider/discovery/bundle/bundle.d/receivers/postgresql.discovery.yaml.tmpl

```diff
@@ -1,36 +1,19 @@
-{{ receiver "smartagent/postgresql" }}:
+{{ receiver "postgresql" }}:
```

Comment:
rather than replace this file I'd add a new one and set the `enabled: false` for the SA receiver to allow a bw-compatible path for at least a release.

Reply:
Want me to add the bundled_test back in for the smartagent postgresql receiver also in that case?

Reply:
Added back as `enabled: false` let me know if you see anything else you'd like me to address. :D